src/ai/backend/client/cli/app.py

```diff
@@ -138,7 +156,11 @@ def __init__(self, api_session, session_id, app_name,
               help='The application-level protocol to use.')
 @click.option('-b', '--bind', type=str, default='127.0.0.1:8080', metavar='[HOST:]PORT',
               help='The IP/host address and the port number to bind this proxy.')
-def app(session_id, app, protocol, bind):
+@click.option('--arg', type=str, multiple=True, metavar='"--option <value>"', help='Add additional argument when starting service.')
+@click.option('--env', type=str, multiple=True, metavar='"ENVNAME=envvalue"', help='Add additional environment variable when starting service.')
+@click.option('--show-configurables', is_flag=True, 
+              help='List available additional arguments and environment variables when starting service.')
```

Comment:
It is a good idea to have an option to show what's possible.
How about separate this as another command instead of making it as an option?
(e.g., `backend.ai apps mysession`)